socialhome/start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ "$PROJECT" == "socialhome" ]; then
+  # fetch user repo
+  cd /socialhome \
+    && git remote add custom $PRREPO \
+    && git fetch custom \
+    && git checkout $PRSHA \
+    && git log -1 || {
+      echo "Cannot find $PRREPO $PRSHA"
+      exit 1
+    }
+  # re-install dependencies
+  cd /socialhome \
+    && pip-sync dev-requirements.txt \
```

Comment:
Oh one thing. You should also `npm i` since there might be new node dependencies in the branch.

Reply:
alright! will do that. haven't tested it yet. the dudes here want me to play WCIII so I have to finish it tomorrow :P

Reply:
The `npm i ` would be good to add here still. PR's frequently add new node modules. Not that they should matter for federation tests, but who knows. `npm run dev` might fail or some side effect.